pipelines/migration/projeto_subsidio_sppo/constants.py

```diff
@@ -709,3 +710,289 @@ class constants(Enum):  # pylint: disable=c0103
+            },
+            "dbt_utils.accepted_range__km_subsidiada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_subsidiada_faixa maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__pof__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna pof maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_apurada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_apurada_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_subsidiada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_subsidiada_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_planejada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_planejada_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__media_pof__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna media_pof maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__valor_a_pagar__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna valor_a_pagar maiores ou igual a zero"
+            },
+            "dbt_utils.unique_combination_of_columns__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as linhas de sumario_faixa_servico_dia_pagamento são unicas"
+            },
+            "dbt_expectations.expect_row_values_to_have_data_for_every_n_datepart__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as datas possuem dados"
+            },
+            "check_km_planejada__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as viagens possuem km_planejada correspondente a OS"
+            },
+            "teto_pagamento_valor_subsidio_pago__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos serviços abaixo do teto de pagamento de valor do subsídio"
+            },
+            "dbt_expectations.expect_table_aggregation_to_equal_other_table__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos serviços com valores de penalidade aceitos"
+            },
+            "sumario_servico_dia_tipo_soma_km__km_apurada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as somas dos tipos de quilometragem são equivalentes a quilometragem total"
+            },
+            "expression_is_true__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as somas de valor_a_pagar e valor_penalidade não nulos e maior ou igual a zero"
```

Comment:
```suggestion
                "description": "Todas as somas de `valor_a_pagar` e `valor_penalidade` não nulas e maiores ou iguais a zero"
```
Alterar para todos os casos semelhantes